processing/src/main/java/org/apache/druid/indexer/TaskLocation.java

```diff
@@ -30,21 +31,29 @@
 
 public class TaskLocation
 {
-  private static final TaskLocation UNKNOWN = new TaskLocation(null, -1, -1);
+  private static final TaskLocation UNKNOWN = new TaskLocation(null, -1, -1, null);
 
   @Nullable
   private final String host;
   private final int port;
   private final int tlsPort;
 
+  @Nullable
+  private final String k8sPodName;
```

Comment:
Should we use a more generic field name here instead of named it after `k8s`? Maybe just name it `jobName`

Reply:
hmm, is there another use case for using this field? it doesn't show up in the console if its not set so i figured its okay to leave it as more specific

Reply:
I think `k8sJobName` will be a good name here as this is what it is called in the code in `K8sTaskId`